content/200-concepts/050-overview/300-prisma-in-your-stack/03-is-prisma-an-orm.mdx

```diff
@@ -1,215 +1,157 @@
+
+Because databases usually contain data, migrations assist you with breaking down schema changes into smaller units which helps avoid inadvertent data loss.
+
+Assuming you were starting a project from scratch, this is what a full workflow would look like: you create a migration that will create the `User` table in the database schema and define the `User` entity class as in the example above.
+
+Then, as the project progresses and you decide you want to add a new `salutation` column to the `User` table, you would create another migration which would alter the table and add the `salutation` column.
+
+Let's take a look at a TypeORM migration, will be used.
+In this example, the migration adds a new `salutation` column to the existing `User` table:
+
+```ts
+import { MigrationInterface, QueryRunner } from 'typeorm'
+
+export class UserRefactoring1604448000 implements MigrationInterface {
+  async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`ALTER TABLE "User" ADD COLUMN "salutation" TEXT`)
   }
-  // Model instance method
-  getFullName() {
-    return [this.firstname, this.lastname].join(' ')
+
+  async down(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`ALTER TABLE "User" DROP COLUMN "salutation"`)
   }
 }
-
-User.init(
-  {
-    id: {
-      allowNull: false
-      autoIncrement: true,
-      primaryKey: true,
-      type: Sequelize.INTEGER,
-    }
-    firstName: {
-      type: Sequelize.STRING,
-      field: 'first_name',
-      allowNull: true
-    },
-    lastName: {
-      type: Sequelize.STRING,
-      field: 'last_name',
-      allowNull: true
-    },
-    email: {
-      type: Sequelize.STRING,
-      allowNull: false,
-      unique: true,
-      validate: {
-        isEmail: true
-      }
-    },
-    emailConfirmed: {
-      type: Sequelize.BOOLEAN,
-      field: 'email_confirmed',
-      allowNull: false,
-      defaultValue: false
-    },
-    birthDate: {
-      type: Sequelize.DATE,
-      field: 'birth_date'
-    }
-  },
-  { sequelize }
-)
 ```
 
-Model instances represent database records and contain three important things:
+> **Note:** TypeORM also offers the [`synchronize`](https://github.com/typeorm/typeorm/blob/master/docs/faq.md#how-do-i-update-a-database-schema) option which updates the database schema based on your entity classes. However, it is not recommended for use in production as it gives little flexibility how changes are carried out and can cause inadvertent data loss. Therefor it isn't addressed in this section.
```

Comment:
Is this note really necessary?

Reply:
I think it is as it maps to our new `prisma db push` and is not considered to be a migration solution.